File: Lecture-004/25_Pattern.cpp
#include <iostream>
using namespace std;

int main() {
    /*
          1
        2 3
      4 5 6
    7 8 9 10
    */

    int n;

    cout<<"Enter the value of n : ";
    cin>>n;

    int row = 1;
    int count = 1;

    while(row <= n) {
        int col = 0;

        while(col < n-row) {
            cout<<"  ";
            col = col + 1;
        }

        while(col < n) {
            cout<<count<<" ";
            count = count + 1;
            col = col + 1;
        }

        cout<<endl;
        row = row + 1;
    }
}

File: Lecture-004/Python Code/25_Pattern.py
"""
          1
        2 3
      4 5 6
    7 8 9 10
"""

n = int(input("Enter the value of n : "))
count = 1
for i in range (0, n):
    for j in range (1, n-i):
        print(end="  ")
    for j in range (0, i+1):
        print(count, end=" ")
        count += 1
    print()